components/azure-iot/README.md

```diff
@@ -0,0 +1,89 @@
+
+# Get Started with Microsoft Azure IoT Starter Kit - ESP32-DevKitC ("Core Board")
+
+This sample was tested with Espressif's **ESP32-DevKitC ("Core Board")**, but many other kits would work as well.
+
+Don't have a kit yet? Click [here](http://esp32.net/)
+
+This sample was modified from [this one](https://github.com/ustccw/AzureESP32.git).
+
+## Step 1 - Download the ESP32 SDK
+
+Clone the [Espressif IoT Development Framework](https://github.com/espressif/esp-idf) repository with the following command:
+
+`git clone https://github.com/espressif/esp-idf.git --recursive`
+
+## Step 2 - Set up the ESP32 toolchain
+
+Follow the instructions for setting up the ESP32 toolchain found [here](http://esp-idf.readthedocs.io/en/latest/#setup-toolchain).
+
+## Step 3 - Set IDF_PATH
+
+Set the IDF_PATH environment variable to point to the location of the **esp-idf** directory that you 
+cloned in Step 1.
+
+If you're using MSYS on Windows, a good place to set the IDF_PATH variable is in the 
+`~\msys32\home\user\.bashrc` file that gets created the first time you run `msys2_shell.cmd`.
+
+## Step 4 - Assemble the sample project
+
+Your `$IDF_PATH/components/azure-iot/sample` directory contains a sample project skeleton. You may 
```

Comment:
Examples for ESP-IDF components are normally located under '$IDF_PATH/examples' directory. Could you please move the sample into `examples/protocols/azure-iot`?